.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
One question though, what's this statement supposed to do?

Reply:
Nevermind.

> If set to true, then npm will stubbornly refuse to install (or even consider installing) any package that claims to not be compatible with the current Node.js version.

Reply:
It's actually something else for our case. Otherwise `npm install` won't error with the wrong npm version.